File: python/010.py
# Summation of Primes

import math

def is_prime(n):
    if n <= 1:
        return False
    
    for i in range(2, int(math.sqrt(n)) + 1):
        if n % i == 0:
            return False
    
    return True

print(sum([i for i in range(1, 2000000) if is_prime(i)]))

File: c++/010.cpp
// Summation of Primes

#include <iostream>

using namespace std;

int isPrime(int number)
{
    if (number <= 1)
        return false;
    
    for (int i = 2; i <= number / 2; i++)
        if (number % i == 0)
            return false;
    
    return true;
}

int main()
{
    long sum = 0;

    for (int i = 0; i < 2000000; i++)
    {
        if (isPrime(i))
            sum += i;
    }

    cout << sum;
}
